File: 02_DevApplication_IOS_avecObjectif-C/10_UITableView_CellulePerso_	MasterDetails/10_UITableView_CellulePerso_	MasterDetails/Gateau.m
//
//  Gateau.m
//  10_UITableView_CellulePerso_	MasterDetails
//
//  Created by etudiant on 21/04/2017.
//  Copyright © 2017 etudiant. All rights reserved.
//

#import "Gateau.h"

@implementation Gateau
-(id)initNom:(NSString*)nom nomImage:(NSString*)nomImage description:(NSString*)description
{
    if(self = [super init])
    {
    self.nomImage = nomImage;
    self.nom = nom;
    self.texte= description;
    }
    return self;
}
@end

File: 03_Swift_fondamentaux/06_UITableView_CelllutePersonnalis"e/06_UITableView_CelllutePersonnalis"e/Gateau.swift
//
//  Gateau.swift
//  06_UITableView_CelllutePersonnalis"e
//
//  Created by etudiant on 11/05/2017.
//  Copyright © 2017 etudiant. All rights reserved.
//

import Foundation

class Gateau
{
    var nomImage: String
    var nom : String
    var description : String
    init(nom:String,nomImage:String,description:String)
    {
         self.nom = nom
        self.description = description
        self.nomImage = nomImage
    }
}
